### PR TITLE
implement select screen

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot from 2023-05-10 11-57-00

This is the select account screen in the wallet onboarding interface

### CODE REVIEW

File: web-frontend/src/pages/Welcome.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import LandingPage from '../layouts/LandingPage/LandingPage';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
```
Comment (Thykof) [inline web-frontend/src/pages/Welcome.tsx:2]: imported but not used
```suggestion
```

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      console.log('clicked');
+    },
+    preIcon: <FiPlus className="w-6 h-6" />,
+  };
+
+  const margin24 = 'mb-6';
+  const margin16 = 'mb-4';
+  const defaultFlex = 'flex flex-col justify-center items-center align-center';
+  const buttonFontColor = 'text-neutral';
+  const selectAccountWidth = 'w-[498px]';
```
Comment (Thykof) [inline web-frontend/src/pages/SelectAccount/SelectAccount.tsx:26]: I don't see the interest of margin24, margin16, buttonFontColor, selectAccountWidth

**Review by Thykof — CHANGES_REQUESTED** [review]

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx (hunk 2 above)
Comment (Thykof) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:26]: WDYT @mazmassa

Comment (mazmassa) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:26]: Agree!

I would say if we have only related we can place it directly to the html tag. Only if we have something bigger or with events like: hover, etc...

Comment (mazmassa) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:26]: BTW, why we are setting that in 498px!? 🤔  Why not 50% or something related!? w-1/2!?

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          <div id="account-select" className="w-full flex flex-col">
+            <div className={margin16}>
+              <AccountSelectorButton {...selectProps} />
+            </div>
+            <div className={margin16}>
+              <AccountSelectorButton {...selectProps} />
+            </div>
+            <div>
+              <Button variant="secondary" {...buttonProps}>
+                {' '}
```
Comment (mazmassa) [inline web-frontend/src/pages/SelectAccount/SelectAccount.tsx:56]: 🤔

File: web-frontend/src/pages/Welcome.tsx (hunk 1 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Welcome.tsx]: Changed !

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx (hunk 2 above)
Comment (pivilartisant) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:26]: Yes ! I wanted to sync with you to see how we change that ! thanks for the suggestion !

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx (hunk 3 above)
Comment (pivilartisant) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:56]: it's the linter !

**Review by peterjah — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**